core/src/main/java/nl/nn/adapterframework/unmanaged/DefaultIbisManager.java

```diff
@@ -462,4 +465,13 @@ public void setApplicationEventPublisher(ApplicationEventPublisher applicationEv
 	public ApplicationEventPublisher getApplicationEventPublisher() {
 		return applicationEventPublisher;
 	}
+
+	@Override
+	public void afterPropertiesSet() throws Exception {
+		boolean requiresDatabase = AppConstants.getInstance().getBoolean("jdbc.required", true);
+		if(requiresDatabase) {
```

Comment:
requiresDatabase hoeft niet te betekenen dat er ook een transactiemanager moet zijn.

Reply:
Nee klopt maar die is er standaard vanwege Spring, de IbisManager heeft geen datasource maar wel een transactiemanager.

Deze zelfde check werd eerst door de scheduler gedaan om te controlleren of er een database was.